configuration/pipeline_config.json

```diff
@@ -0,0 +1,53 @@
+{
+  "PipelineName": "WorldVision",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/world-vision-textit-token.txt",
+      "ContactsFileName": "contacts",
+      "ActivationFlowNames": [
+        "worldvision_s01e01_activation"
+      ],
+      "SurveyFlowNames": [
+        "worldvision_s01_demog"
+      ],
+      "TestContactUUIDs": [
+        "3c2d94f4-3833-4828-a577-cd6c082135df",
+        "015c8230-23b3-4207-b961-ebcc7acc1da9"
+      ]
+    }
+  ],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "WorldVision_phone_number_avf_phone_id"
+  },
+  "RapidProKeyRemappings": [
+    {"RapidProKey": "avf_phone_id", "PipelineKey": "uid"},
+
+    {"RapidProKey": "Rqa_S01E01 (Text) - worldvision_s01e01_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E01 (Run ID) - worldvision_s01e01_activation", "PipelineKey": "rqa_s01e01_run_id"},
+    {"RapidProKey": "Rqa_S01E01 (Time) - worldvision_s01e01_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Constituency (Text) - worldvision_s01_demog", "PipelineKey": "location_raw"},
```

Comment:
do we intend to copy over the demog responses from ke covid19 projects?

Reply:
Good question. I've checked with the team and we've decided no because:
 - This creates a lot of additional effort for what is only ~120 people (of who only a few will respond)
 - We'd need to be careful about what we copy across, because we don't have permission to share project data in the same way we do for CSAP.
 - We only asked 3 questions here and ke-urban, so the burden on participants is relatively low.